packages/policy/.prettierignore

```diff
@@ -0,0 +1,6 @@
+**/node_modules
+**/coverage
+test/fixtures
+build/
+docs/
+protos/
```

Comment:
```suggestion
protos/

```

Reply:
This came from the generate-bot script.  Will update it along with the feedback above after this lands.